tests/album-read.test.js

```diff
@@ -0,0 +1,61 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Read Albums', () => {
+  let albums;
+  beforeEach(async () => {
+    const responses = await Promise.all([
```

Comment:
similar to what I mentioned in the album-delete test file, add an artist first then an album and link them